modules/plot/main/coffee/Expose.coffee

```diff
@@ -7,6 +7,14 @@ hx.plot.arcCurve = arcCurve
 hx.plot.svgCurve = svgCurve
 hx.Axis = Axis
 hx.Graph = Graph
+
+hx.graph = (options) ->
+  selection = new hx.detached 'div'
+  graph = new Graph selection.node(), options
+  # There is no point rendering it now, the selection is of zero size.
+  selection.on 'resize', -> graph.render()
```

Comment:
Plot already adds a resize listener on line 28 of Graph.coffee in the hx.Graph constructor so I don't think the on-resize is needed?
```
selection = hx.select(@selector).on 'resize', 'hx.plot', => @render()
```

Also, when adding event listeners inside hexagon, you should namespace them in the format `hx.{module-name}`, e.g. `hx.plot`, `hx.data-table` etc